assignment/tool/src/webapp/vm/assignment/chef_assignments_instructor_grading_submission.vm

```diff
@@ -221,31 +221,22 @@
         </table>
 
         ## show or hide assignment instruction and attachment
-        #if ($assignment_expand_flag)
+        <div id="assignamentInstructionPanel">
             <p class="discTria">
-                <a href="javascript:void(0)" onclick="ASN.submitForm( 'gradeForm', 'hide_instruction', null, null ); return false;">
-                    <img src="#imageLink("sakai/collapse.gif")" alt="$tlang.getString("gradingsub.opecliass")" border="0" width="13" height="13" align="top"/>
+                <a href="javascript:void(0)" onclick="ASN.toggleAssignamentInstructionPanel();">
+                    <img id="expandAssignamentInstruction" src="#imageLink("sakai/expand.gif")" alt="$tlang.getString("gen.clocli")" border="0" width="13" height="13" align="top"/>
+                    <img id="collapseAssignamentInstruction" style="display: none;" src="#imageLink("sakai/collapse.gif")" alt="$tlang.getString("gradingsub.opecliass")" border="0" width="13" height="13" align="top"/>
+                    $tlang.getString("gen.assinf")
                 </a>
-                $tlang.getString("gen.assinf")
             </p>
-        #else
-            <p class="discTria">
-                <a href="javascript:void(0)" onclick="ASN.submitForm( 'gradeForm', 'show_instruction', null, null ); return false;">
-                    <img src="#imageLink("sakai/expand.gif")" alt="$tlang.getString("gen.clocli")" border="0" width="13" height="13" align="top"/>
-                </a>
-                $tlang.getString("gen.assinf")
-            </p>
-        #end
-        #if ($assignment_expand_flag)
```

Comment:
is assignment_expand_flag no longer needed?

Reply:
We don't need it now.

Reply:
OK Great can we remove it from other places as well so that it is no longer needed?

Reply:
The flag is used in chef_assignments_student_review_edit.vm, too. Maybe you could use the same approach as here in that file, and then get rid of the context attribute in AssignmentAction?